### DISCUSSION TITLE
how to pay for the Ai while in iran?

### DISCUSSION THREAD

**Discussion by blackhawk06660** [community/community#134005] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hello, im not sure if its the right place to post it but i need access to the ai for personal Arduino based projects and to tweak some anti circumvention tools. not only 10 dollars is not cheap based on my budget (yeah i know !!!, but trust me. how can a countries currency be this messed up?) i cant pay because i have no access to credit cards such as master or visa. any ideas how to do it ? most probably even our educational facilities are not verified so that i would be able to apply for a free student access.

**Comment by std-enigma** [comment]
https://license-market.ir/product/Github-Copilot

**Comment by std-enigma** [comment]
will you please mark this as an answer if this helped you

**Comment by baharseyd** [comment]
some company in Iran can help u to open an account  in other country

**Comment by std-enigma** [comment]
just check [this](https://license-market.ir) shop

**Reply by Ai00age** [reply]
fake site ?

**Reply by std-enigma** [reply]
Why would you think that?

**Reply by blackhawk06660** [reply]
a friend did ask them a while ago, was nt happy with some of them services.
